effects.cwt

```diff
@@ -981,15 +981,16 @@ alias[effect:remove_consort] = character
 alias[effect:remove_favor] = character
 
 ## scope = character
-###
+# TODO: Confirm what scopes this applies to
+### Removes a friend relationship between the scoped and the given characters
 alias[effect:remove_friend] = character
```

Comment:
Basically what you want to do is change the right hand side here to scope[character]. That will mean that remove_friend takes any scope that refers to a character. In other cases, you might want to use <character> - this'll mean specifically a character ID and no other way to refer to a character. This is the gist of what needs to be done about most effects; figure out whether it's a scope or an absolute reference and adjust RHS accordingly.